### DISCUSSION TITLE
Slow UI for one specific App

### DISCUSSION THREAD

**Discussion by Mais316** [argoproj/argo-cd#14311] [Q&A]
Hi All,
We have ArgoD 2.5.6, with one parent application that auto create AppSet/project resources per customer, so in brief one mono repository that has a directory per customer to create dedicated AppSet and Project after onboarding more than 100+ customer the parent app started to be very slow to load and edit via UI.

ArgoCD UI is totally fine loading any other app with exception to parent app, do I need to add ".argocd-allow-concurrency" to the parent app mono repo main path for now path is "./" and it has a directory per customer, for example dir x has applicationset.yaml and project.yaml for this customer then dir y same, ....

We have 4 argocd-server replicas and two argocd-controllers, 2 argocd-repo-server, one applicationset-controller.

**Comment by Mais316** [comment]
please help

**Reply by crenshaw-dev** [reply]
Can you describe in more detail what the UI being slow means? Ideally it would be helpful to see a network call history and a frontend performance profile.
